### ISSUE TITLE
feat: notice for apprenticeships that are currently on pause (wip)

### ISSUE THREAD

**Issue by kaseybon** [sparkbox/apprentice-at#295]
**Description**
Because of the current COVID-19 pandemic some apprenticeships are on hold. This card is to add a notice to each affected apprenticeship with the following code. Each notice should include the name of the company that links to the company homepage.

**Design**
<img width="887" alt="Apprentice_at" src="https://user-images.githubusercontent.com/2486846/95230713-3c3aa380-07d0-11eb-8d39-ebad96375b83.png">

`padding: 10px 20px;`
`background-color: $c--iceberg;`

**Out of Scope**
We will not be updating any listings within this card so there should be no visible changes to the site once the work is merged. This card should only give us the ability to include a notice.

**Acceptance Criteria**
- Notices can be turned on in the `source/data/data.yml` file by including `covid_notice: true`. Listings without this flag will not show a notice.
- Notices will dynamically include a company name that is linked to the company home page.
- Links within the notice are styled with an underline to meet [accessibility standards](https://webaim.org/techniques/hypertext/link_text#underlining).
